### ISSUE TITLE
api.RTCStatsReport.type_inbound-rtp.nackCount - reported by at least Safari for iOS 17.1.2

### ISSUE THREAD

**Issue by david-flok** [mdn/browser-compat-data#22293]
### What type of issue is this?

Browser bug (a bug with a feature that may impact site compatibility)

### What information was incorrect, unhelpful, or incomplete?

`nackCount` is reported by at least Safari for iOS 17.1.2 and probably other browsers at older versions.

### What browsers does this problem apply to, if applicable?

Safari

### What did you expect to see?

If support status is unknown or hasn't been captured, I would expect the table to reflect that instead of definitively saying the feature is unsupported.

### Did you test this? If so, how?

Yes, I reviewed the output of the stats API reported by a Safari for iOS 17.1.2 client.

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/RTCInboundRtpStreamStats/nackCount

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `api.RTCStatsReport.type_inbound-rtp.nackCount`
* Report started: 2024-02-26T16:08:40.271Z

</details>

**Comment by david-flok** [comment]
Also on Chrome Mobile WebView 121-ish:

> Mozilla/5.0 (Linux; Android 14; SM-G998B Build/UP1A.231005.007; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/121.0.6167.178 Mobile Safari/537.36

**Comment by hamishwillee** [comment]
@queengooborg BCD collection is giving "support unknown" for everything in https://mdn-bcd-collector.gooborg.com/tests/api/RTCStatsReport so impossible to verify this on that tool. Ideally the collector text gets updated. This leaves me concerned about some of the other data too.

I have tested on https://wpt.live/webrtc-stats/supported-stats.https.html and it is giving a pass on this on Chrome 121 and Firefox 123 so the data is definitely bogus.

**Comment by queengooborg** [comment]
Due to the number of issues with testing RTCStatsReport, I've disabled them in the collector for the time being, until we can improve the tests to where they're as reliable as the ones on web-platform-tests.

**Comment by hamishwillee** [comment]
So we'd have to bisect to find the supported versions?

**Comment by hamishwillee** [comment]
@david-flok I don't think this issue has changed. The problem is indeed data quality. Normally this would populate from the BCD collector, but that's broken for this API. We don't use "unknown" in  this repo, so I'm querying whether the only way to test this is to bisect versions using the web test.

**Comment by hamishwillee** [comment]
> I'm curious about why 'unknown' isn't a possible result here.

So historically you could record `null` (unknown), `true`, `false`, version start/end. Data quality was not great because people always just put null in for all cases they weren't interested in. By not allowing it we know that the overall quality has become very good for web APIs.

As an aside, I'm not in the BCD team (I'm in MDN content). But I lived through the change at things are much more accurate now than they were.

**Comment by hamishwillee** [comment]
I can't test Safari, but have at least updated chrome and FF in #22342
